x-pack/legacy/plugins/infra/public/components/logging/log_text_stream/scrollable_log_text_stream_view.tsx

```diff
@@ -52,7 +52,7 @@ interface ScrollableLogTextStreamViewProps {
   }) => any;
   loadNewerItems: () => void;
   reloadItems: () => void;
-  setFlyoutItem: (id: string) => void;
+  setFlyoutRef: (ref: { id: string; index: string }) => void;
```

Comment:
There's two setFlyoutItem in code base so this made sense. One is setting the loaded item. This one setting the one we want to load. So made sense to rename for two reason, it now takes more than just ID and it takes reference not an item